main.css

```
@@ -2,4 +2,15 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+body{
```

🙃 Make sure to leave some space for your declarations
```suggestion
body {
```